src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw

```diff
@@ -814,21 +814,13 @@
   <data name="ColorScheme_RenameErrorTip.Title" xml:space="preserve">
     <value>This color scheme name is already in use.</value>
   </data>
-  <data name="SettingContainer_OverrideIntro" xml:space="preserve">
-    <value>This overrides the value in {}.</value>
-    <comment>"{}" is the object being overridden. Generally, it will be the name of another profile.</comment>
-  </data>
-  <data name="SettingContainer_ResetButtonHelpText" xml:space="preserve">
-    <value>Reset</value>
-  </data>
-  <data name="SettingContainer_OverrideTarget" xml:space="preserve">
-    <value>a lower layer</value>
-    <comment>This is the object of "SettingContainer_OverrideIntro".</comment>
-  </data>
   <data name="Globals_FocusFollowMouse.Header" xml:space="preserve">
     <value>Enable Focus Follow Mouse mode</value>
   </data>
   <data name="Globals_FocusFollowMouse.HelpText" xml:space="preserve">
     <value>When checked, the terminal will the focus pane on mouse hover.</value>
   </data>
-</root>
+  <data name="SettingContainer_OverrideMessageBaseLayer" xml:space="preserve">
+    <value>Reset to base layer value</value>

```

Comment:
nit: add a comment explaining to the translator that this "base layer" is a reference to the Base Layer key above (whatever its name is)!